src/BloomExe/MiscUI/ProblemReporterDialog.cs

```diff
@@ -148,9 +150,22 @@ private void UpdateDisplay(object sender, EventArgs e)
 #endif
 		}
 
+		public bool IsLegalEmail(string emailAddress)
+		{
+				//from http://stackoverflow.com/a/6893571/723299
+				return Regex.IsMatch(emailAddress, @"^[\w!#$%&'*+\-/=?\^_`{|}~]+(\.[\w!#$%&'*+\-/=?\^_`{|}~]+)*"
+					+ "@"
+					+ @"((([\-\w]+\.)+[a-zA-Z]{2,4})|(([0-9]{1,3}\.){3}[0-9]{1,3}))$"); 
+		}
+
 		protected virtual void UpdateDisplay()
 		{
-			_submitButton.Enabled = !string.IsNullOrWhiteSpace(_name.Text.Trim()) && !string.IsNullOrWhiteSpace(_email.Text.Trim()) &&
+			if(!string.IsNullOrWhiteSpace(_email.Text.Trim()))
+			{
+				_email.ForeColor = IsLegalEmail(_email.Text) ? Color.Black : Color.Red;
+			}
+
+			_submitButton.Enabled = !string.IsNullOrWhiteSpace(_name.Text.Trim()) && !string.IsNullOrWhiteSpace(_email.Text.Trim()) && IsLegalEmail(_email.Text) &&
```

Comment:
Do we definitely not want a problem report from a rice farmer who doesn't have his own email? I suppose it's unlikely to have internet and not email...seemed worth asking, though.

Will it be sufficiently obvious to a rice farmer that he has to fill in all three fields before submit is enabled? It's good to show email in red if invalid, but I prefer the design where compulsory fields are highlighted in some way when the (not disabled) submit button is clicked.